package.json

```
@@ -1,15 +1,22 @@
 {
   "name": "rollup-plugin-obfuscator",
-  "version": "0.2.0",
+  "version": "0.2.1",
   "description": "The most powerful rollup plugin for javascript-obfuscator",
   "main": "rollup-plugin-obfuscator.js",
+  "scripts": {
+    "build": "tsc rollup-plugin-obfuscator.ts"
```

Could you add the following to ensure I don't forget to compile before publishing to NPM?

```suggestion
    "build": "tsc rollup-plugin-obfuscator.ts",
    "prepare": "npm run build"
```